code/game/objects/items.dm

```diff
@@ -696,6 +696,14 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 		to_chat(M, SPAN_WARNING("You are too distracted to look through the [devicename], perhaps if it was in your active hand this might work better."))
 		cannotzoom = 1
 
+	if(ishuman(M)) //this code is here to stop species night vision from being used on the cameras, since it does not make sense since cameras are just images. this is probably not the best way to do this, but it works
+		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/E = H.internal_organs_by_name[BP_EYES]
+		if (istype(E, /obj/item/organ/internal/eyes/night))
+			var/obj/item/organ/internal/eyes/night/N = E
+			if(N.night_vision )
+				N.disable_night_vision()
```

Comment:
This should all be consolidated into a human or mob level proc.

Reply:
Solved.